esp_delta_ota/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+idf_component_register(SRCS "src/esp_delta_ota.c" "detools/c/detools.c" "detools/c/heatshrink/heatshrink_decoder.c"
+                       INCLUDE_DIRS "include" 
+                       PRIV_INCLUDE_DIRS "detools/c" "detools/c/heatshrink")
+
+target_compile_options(${COMPONENT_LIB} PRIVATE "-DDETOOLS_CONFIG_COMPRESSION_LZMA=0")
```

Comment:
Do we really need this compile option?

Reply:
Yes. Without this, `esp_delta_ota` component tries to include `lzma.h` which causes compile error if it is not present. Also we are not using `lzma` compression.